test/test_multi_region_cubed_sphere.jl

```diff
@@ -7,6 +7,30 @@ using Oceananigans.BoundaryConditions: fill_halo_regions!
 using Oceananigans.Fields: replace_horizontal_vector_halos!
 using Oceananigans.Models.HydrostaticFreeSurfaceModels: fill_velocity_halos!
 
+function read_big_endian_coordinates(filename, nInterior = 32, nHalo = 1)
```

Comment:
```suggestion
function read_big_endian_coordinates(filename, Ninterior = 32, Nhalo = 1)
```

Always good to use consistent notation with Oceananigans

Reply:
Incorporated in this [commit](https://github.com/CliMA/Oceananigans.jl/pull/3306/commits/8351eecbe7b067d18240a133504be167d0d24d7b).